index.js

```diff
@@ -1,68 +1,79 @@
 module.exports = {
     env: {
-        'jasmine': true
+        es6: true,
+        jasmine: true,
+        node: true
     },
-
-    // set this as the base `.eslint` file
-    root: true,
-
     // start with a sane baseline
     extends: 'eslint:recommended',
-
+    parserOptions: {
+        ecmaVersion: 9,
+        sourceType: 'module',
+    },
     plugins: ['jasmine'],
-
-    // linting rules settings
+    // set this as the base `.eslint` file
+    root: true,
     rules: {
         'array-bracket-spacing': ['error', 'never'],
         'block-spacing': ['error', 'always'],
         'brace-style': ['error', '1tbs', {
-            'allowSingleLine': true
+            allowSingleLine: true
         }],
-        'camelcase': ['error', { 'properties': 'never' }],
+        camelcase: ['error', { properties: 'never' }],
         'comma-spacing': ['error', {
-            'before': false, 'after': true
+            before: false, after: true
         }],
         'comma-style': ['error', 'last'],
+        curly: ['error', 'all'],
         'eol-last': ['error', 'always'],
-        'indent': ['error', 4, {
-            'MemberExpression': 0,
-            'SwitchCase': 1,
-            'VariableDeclarator': 1,
-            'outerIIFEBody': 1,
-            'FunctionDeclaration': {
-                'parameters': 1,
-                'body': 1
+        'global-require': 'error',
+        indent: ['error', 4, {
```

Comment:
2?

Reply:
2

Reply:
Frontend does 2, backend does 4?

Reply:
I think 2 is good to be consistent.

Reply:
2!

Reply:
happy to go with 2, but will set it as a "warn" initially so that it doesn't break all the builds